Description: Added HamoniKR 4.0 ASC II logo
 .
 neofetch (7.0.0-1+hamonikr1) unstable; urgency=medium
 .
   * Added HamoniKR ASC II Art
Author: HamoniKR <pkg@hamonikr.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2020-12-29

--- neofetch-7.0.0.orig/neofetch
+++ neofetch-7.0.0/neofetch
@@ -7078,7 +7078,33 @@ ${c2}          :dc'
                         ${c1}
 EOF
         ;;
-
+        
+        "HamoniKR"*)
+            set_colors 4 7
+            read -rd '' ascii_data <<'EOF'
+${c1}            
+                   _.                   
+                _uS2Sos.                
+              <uXSo2o2os                
+           .<o22oo2o2o2(                
+         .<no2o2o2oooe`                 
+       .%no2oooooooo}                   
+     .vnoooooooooone                    
+   .=i|=====+"!vnnn(                    
+ .|%vvIvvvvvI~  "{n(           .;|||=_  
+.IvIIIIvIvl^      +n,        .=|||+|+|; 
+ vlIvIII%~         <l,     _=||||+|+||; 
+  "|ii>~            vvs,  =||||+|+|=+-  
+                   .vvlll|_==;;====-    
+                   =llllIIvvllll>`      
+                  _llllllliiil+`        
+                .=Illlliliil+`          
+               .illiiiiiii+`            
+               -iiiiiiii~               
+                -+iii>~                                                                                                 
+EOF
+        ;;
+        
         "Huayra"*)
             set_colors 4 7
             read -rd '' ascii_data <<'EOF'
@@ -10019,33 +10045,6 @@ oMMMMmMMMMNds:.+MMMmmMMN/.-odNMMMMmMMMM+
              .odNm+  /dNms.
 EOF
                 ;;
-        
-        "HamoniKR"*)
-            set_colors 4 7
-            read -rd '' ascii_data <<'EOF'
-${c1}            
-                   _.                   
-                _uS2Sos.                
-              <uXSo2o2os                
-           .<o22oo2o2o2(                
-         .<no2o2o2oooe`                 
-       .%no2oooooooo}                   
-     .vnoooooooooone                    
-   .=i|=====+"!vnnn(                    
- .|%vvIvvvvvI~  "{n(           .;|||=_  
-.IvIIIIvIvl^      +n,        .=|||+|+|; 
- vlIvIII%~         <l,     _=||||+|+||; 
-  "|ii>~            vvs,  =||||+|+|=+-  
-                   .vvlll|_==;;====-    
-                   =llllIIvvllll>`      
-                  _llllllliiil+`        
-                .=Illlliliil+`          
-               .illiiiiiii+`            
-               -iiiiiiii~               
-                -+iii>~                                                                                                 
-EOF
-        ;;
-
             esac
         ;;
     esac
